Fixed configure script compatibility issue.
Applied manually on configure from 92e37cb4d7760bd4a1e6cd07eb61d33f4c402647


--- configure.orig	2025-03-28 23:35:13 UTC
+++ configure
@@ -17588,7 +17588,7 @@ printf "%s\n" "#define API_VERSION 11" >>confdefs.h
     CXX="$CXX"
   else
     result=`echo "${values_to_check}" | $GREP -Fvx -- "${unwanted_values}" | $GREP -v '^$'`
-    CXX=${result//$'\n'/ }
+    CXX=$(echo "$result" | tr -d '\n')
   fi
 
 
